openstates/tx/people.py

```diff
@@ -172,17 +192,18 @@ def _scrape_representative(self, url, term, parties):
 
         # Vacant house "members" are named after their district numbers:
         if re.match(r'^\d+$', scraped_name):
-            return None
+            yield None
```

Comment:
Greg had a concern here, "I'm a little concerned this may blow something up, but there aren't any empty district seats at the moment."